docs/src/user_docs/assembly/extension_field_operations.md

```diff
@@ -0,0 +1,13 @@
+# Extension Field Operations
+Miden assembly provides a set of instructions which can perform operations with extension field elements. These instructions are described in the table below.
+
+### Arithmetic operations
+
+| Instruction | Stack Input | Stack Output | Notes |
+| ----------- | ----------- | ------------ | ----------- |
+| ext2_add <br> - *(5 cycles)*  <br>  | [b, a, ...] | [c, ...] | $c \leftarrow (a + b) \mod p$                                |
```

Comment:
For $(a+b) \mod p$ we should define what p is. You can find the irreducible polynomial https://github.com/0xPolygonMiden/miden-vm/blob/bffed62b3cbdf509dc0a9ba0d789247df3213607/stdlib/asm/math/ext2.masm#L77